OWNERS_ALIASES

```diff
@@ -17,17 +17,11 @@ aliases:
     - vpavlin
     - yanniszark
   wg-manifests-leads:
-    - PatrickXYS
     - StefanoFioravanzo
-    - elikatsis
     - kimwnasptd
-    - yanniszark
   wg-notebooks-leads:
-    - StefanoFioravanzo
```

Comment:
@StefanoFioravanzo is active again within Kueflow

Reply:
@juliusvonkohout thanks for mentioning this. I won't be contributing code anytime soon I think. In quality of release product manager I am contributing to the notebooks wg only in terms of reviewing design docs, helping with the roadmap and aligning with the user feedback we gather.